### ISSUE TITLE
Cannot pass /clp:NoSummary to dotnet build

### ISSUE THREAD

**Issue by dasMulli** [dotnet/cli#7009]
## Steps to reproduce

```
dotnet build /clp:NoSummary
```

## Expected  behavior

No build summary shall be displayed (e.g. duplicated errors).

The behaviour should be the same as using it with msbuild:

```
$ dotnet msbuild /clp:NoSummary
Microsoft (R) Build Engine version 15.3.388.41745 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

Program.cs(5,21): error CS0116: A namespace cannot directly contain members such as fields or methods [/Users/martin.ullrich/tmp/tmp.csproj]
```

## Actual behavior

Build summary is displayed:

```
$ dotnet build /clp:NoSummary
Microsoft (R) Build Engine version 15.3.388.41745 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

Program.cs(5,21): error CS0116: A namespace cannot directly contain members such as fields or methods [/Users/martin.ullrich/tmp/tmp.csproj]

Build FAILED.

Program.cs(5,21): error CS0116: A namespace cannot directly contain members such as fields or methods [/Users/martin.ullrich/tmp/tmp.csproj]
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:01.50
```

This seems to be because the implementation of the `build` verb always appends `/clp:Summary` to the arguments, overriding the passed in arguments.

## Environment data
`dotnet --info` output:

```
.NET Command Line Tools (2.0.0-preview2-006502)

Product Information:
 Version:            2.0.0-preview2-006502
 Commit SHA-1 hash:  07dfded669

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.12
 OS Platform: Darwin
 RID:         osx.10.12-x64
 Base Path:   /usr/local/share/dotnet/sdk/2.0.0-preview2-006502/

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0-preview2-25407-01
  Build    : 40c565230930ead58a50719c0ec799df77bddee9
```

**Comment by dasMulli** [comment]
Context: https://stackoverflow.com/questions/44582185/duplicate-error-messages-in-dotnet-core

**Comment by dmccaffery** [comment]
Note: /clp is ignored by dotnet build even when passing a project argument in .NET SDK 2.0 -- this is not the same issue as #5607

**Comment by dasMulli** [comment]
Root cause is that the build command appends a clp switch after all user arguments, overriding the value passed via command line.

https://github.com/dotnet/cli/blob/8e756edce1573c30c0c5a0708ff323dda0f673fc/src/dotnet/commands/dotnet-build/BuildCommand.cs#L49-L53

**Comment by nguerrera** [comment]
Fixed by #7896
